packages/element/src/react-platform.native.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { AppRegistry } from 'react-native';
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { applyFilters, doAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { cloneElement } from './react';
+
+const render = ( element, id ) =>
+	AppRegistry.registerComponent( id, () => ( propsFromParent ) => {
```

Comment:
Out of curiosity, why not to execute all this logic inside initializeEditor in the @wordpress/edit-post package?

Reply:
I think it's to be able to customize the editor from an outside package. In this case a repo that's importing Gutenberg as a submodule. Correct me if I'm wrong @Tug?

Reply:
No we could totally have this logic in `@wordpress/edit-post` (or in `@wordpress/react-native-editor` as it was the case before in the monorepo branch before this PR).

The idea here is simply to get closer to having `@wordpress/edit-post` work as cross-platform package. If you look at [`edit-post/src/index.native.js`](https://github.com/WordPress/gutenberg/blob/66c0dc7c709fd0a3fe7a794ca56e304eee5abc8c/packages/edit-post/src/index.native.js`) 
you'll see that the init code is much closer to the [web one](https://github.com/WordPress/gutenberg/blob/66c0dc7c709fd0a3fe7a794ca56e304eee5abc8c/packages/edit-post/src/index.js#L138) now.

> Overriding React methods and adding WP hooks might have some performance implications but I guess it’s called only once.

We're not overriding anything though, the `render` function wasn't defined for native prior to this. And yes it's only rendered once on load.

Reply:
It probably doesn't matter as much then now, but I guess `initializeEditor` would be the best place to run all those side effects.